spring-cloud-alibaba-sentinel-gateway/src/main/java/com/alibaba/cloud/sentinel/gateway/GatewayEnvironmentPostProcessor.java

```diff
@@ -0,0 +1,24 @@
+package com.alibaba.cloud.sentinel.gateway;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.env.EnvironmentPostProcessor;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.PropertiesPropertySource;
+
+import java.util.Properties;
+
+/**
+ * Created by zhh on 2019/8/17.
+ */
+public class GatewayEnvironmentPostProcessor implements EnvironmentPostProcessor {
+    private final static String SENTINEL_FILTER_ENABLED = "spring.cloud.sentinel.filter.enabled";
+    private final static String FILTERS_DISABLED = "filters_disabled";
+
+    @Override
+    public void postProcessEnvironment(ConfigurableEnvironment environment, SpringApplication springApplication) {
+        Properties properties = new Properties();
+        properties.setProperty(SENTINEL_FILTER_ENABLED, "false");
+        PropertiesPropertySource propertySource = new PropertiesPropertySource(FILTERS_DISABLED, properties);
```

Comment:
i think it is no need to create a new `PropertiesPropertySource`, refer https://github.com/alibaba/spring-cloud-alibaba/blob/master/spring-cloud-starter-alibaba/spring-cloud-starter-bus-rocketmq/src/main/java/com/alibaba/cloud/bus/rocketmq/env/RocketMQBusEnvironmentPostProcessor.java

Reply:
Thanks. I'll modify it soon.